README.md

```diff
@@ -1,2 +1,25 @@
-# reactcess
+# react-access
 React Context driven role-access for conditional rendering of components.
+
+## Quick Start
+`npm i react-access` or `yarn add react-access`
+
+```
+import RequireForAccess, {ReactAccessContext} from 'react-access';
+import {MyApp, AdminMenuBar} from './my-app';
+
+// you can hydrate these to your app however you'd like on initial page load
+// or in your bundle, etc
+const userPermissions = ['APPUSER', 'CREATE', 'EDIT', 'ETC'];
+
+React.render(<ReactAccessContext userPermissions={userPermissions}>
+  <RequireForAccess permissions={['ADMIN']} invalidAccessComponent={<span>You don't have admin access!</span>}>
```

Comment:
GitHub has a fixed width website so this lines bleeds over and causes a horizontal scrollbar, it might be better to refactor this line to use 8 or fewer characters than currently in it. Maybe spread over multiple lines?